ParquetSharp.Dataset/ParquetSharp.Dataset.csproj

```diff
@@ -15,12 +15,11 @@
     <PackageTags>dataset apache parquet hive partitioned partitioning gresearch g-research .net c#</PackageTags>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/G-Research/ParquetSharp.Dataset</RepositoryUrl>
-    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
 
   <ItemGroup Label="Additional nuget files">
-    <None Include="../LICENSE" Pack="true" PackagePath="\" />
```

Comment:
Hi @thompson-tomo, can you please keep the license file included in the NuGet package

Reply:
hi @adamreeve all done.

Reply:
Thanks!